0.12/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:18.04
```

Comment:
This is a large image to start from.  Why not use `buildpack-deps:bionic-scm` ?

Reply:
A image using buildpack-deps is bigger than using ubuntu.
```
$ sudo docker images
REPOSITORY                 TAG                 IMAGE ID            CREATED             SIZE
tmp2                       latest              d1cc5d0e893c        10 seconds ago      317 MB  # <= FROM buildpack-deps
tmp                        latest              2ca50fa19b8a        5 minutes ago       151.9 MB   #  FROM ubuntu:18.04
docker.io/ubuntu           18.04               20bb25d32758        6 hours ago         87.47 MB
docker.io/buildpack-deps   bionic-scm          4b15e4cfafaf        3 weeks ago         247.9 MB
```

Reply:
I see - my mistake.